data/transactions/verify/txn.go

```diff
@@ -99,42 +99,26 @@ func (g *GroupContext) Equal(other *GroupContext) bool {
 		g.minAvmVersion == other.minAvmVersion
 }
 
-// Txn verifies a SignedTxn as being signed and having no obviously inconsistent data.
```

Comment:
the reason why it was called `Txn` is because it is exported and used as `verify.Txn()`

Reply:
The `func Txn(s *transactions.SignedTxn, txnIdx int, groupCtx *GroupContext) error` is only used in the test, and is moved to 
`func verifyTxn(s *transactions.SignedTxn, txnIdx int, groupCtx *GroupContext) error` in verify/txn_test.go

Here, the code diff is mixing the changes. `TxnBatchVerify` is renamed to `txnBatchVerifyPrep` since it is not actually verifying, but preparing for it, and it is only used inside the package.